Resources/Textures/_RMC14/Objects/Weapons/Guns/Ammunition/Magazines/GSh.rsi/meta.json

```
@@ -0,0 +1,17 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-4.0",
+  "copyright": "taken from directory magazine_smart_gun.rsi  and modified by RavenCaat :3 ",
```

The copyright information needs to include a link to the file with the commit it was taken from
```suggestion
  "copyright": "Taken from https://github.com/RMC-14/RMC-14/tree/e26342c557d307e4eb2a7ad04e3b2c6e4fd56e84/Resources/Textures/_RMC14/Objects/Weapons/Guns/Ammunition/Magazines/SmartGuns/magazine_smart_gun.rsi and modified by RavenCaat :3",
```